src/scss/fullwidth.scss

```diff
@@ -0,0 +1,24 @@
+.fullwidth-split {
```

Comment:
It looks like this is adding new classes over the existing ones, which we don't want to do in a small realease.

Reply:
What do you think about shipping a major release instead? 
There were a lot of changes, small and huge between Bootstrap `v4.0.0-beta.2` and `v4.1`. Please read all the blog posts and focus on the 'Breaking changes' and 'What's new' sections.

📝 https://blog.getbootstrap.com/2017/12/28/bootstrap-4-beta-3/
📝 https://blog.getbootstrap.com/2018/01/18/bootstrap-4/
📝 https://blog.getbootstrap.com/2018/04/09/bootstrap-4-1/

Reply:
Give us a couple of days to think about shipping a major version. We should get with a response early next week regarding a major release.